client/src/components/MediaCard/MediaCard.js

```diff
@@ -0,0 +1,31 @@
+
+import React from 'react';
+import { Link } from "react-router-dom";
+import "./MediaCard.css";
+
+
+// Function based React Component
+const MediaCard = (props) => {
+
+  // Default Class to apply to Component
+  let styles = {
+    height: '248px',
+    width: '248px',
+    backgroundImage: `url(${props.imageSrc})`,
+    backgroundSize: 'cover',
+    display: 'flex',
+    flexDirection: 'column-reverse'
+  }
+  let userPath = `/User/--usernameid--`
+
+  console.log(props.imageSrc)
+  return (
+    <Link className='media-card'to={userPath} style={styles}>
```

Comment:
Instead of passing in all styles here, just pass in the `backgroundImage` as props.